content/hilfe/Handbuch/Mitarbeiterapp2/index.md

```diff
@@ -0,0 +1,231 @@
+---
+title: Mitarbeiter-App2
+weight: 1
+icon: mobile
+---
+
+-- Funktionen der Mitarbeiterapp - Installation separat
+
+
+
+## Einloggen
+
+Nachdem Sie den Mitarbeiter wie unter 
+[Zugänge &Rechte](/hilfe/handbuch/mitarbeiter/zugaenge-rechte) beschrieben per Email
+eingeladen haben und er sich registriert hat, kann er sich nun in der
+Mitarbeiterapp einloggen. Hierzu ruft er die Mitarbeiterapp unter
+https://perso.pentacode.app auf und gibt seine Email Adresse, sowie das
+selbstgewählte Passwort ein. Worauf man in die **Übersicht** der Mitarbeiterapp
+gelangt.
```

Comment:
Ich denke der Artikel sollte generell aus Sicht des Mitarbeiters geschrieben sein. Also bitte aus Sicht des Mitarbeiters beschreiben mit einer Notiz in die Richtung "Muss erst von deinem Arbeitgeber freigeschaltet werden".